packages/wdio-static-server-service/src/launcher.ts

```diff
@@ -1,43 +1,48 @@
-import path from 'path'
+import { join, resolve } from 'path'
 import { promisify } from 'util'
 
 import express from 'express'
 import fs from 'fs-extra'
 import morgan from 'morgan'
 import logger from '@wdio/logger'
+import { FolderOption, MiddleWareOption } from './types'
 
 const log = logger('@wdio/static-server-service')
 
 const DEFAULT_LOG_NAME = 'wdio-static-server-service.log'
 
 export default class StaticServerLauncher {
-    constructor ({ folders, port = 4567, middleware = [] }) {
+    folders: FolderOption[] | null
+    port: number
+    middleware: MiddleWareOption[]
+    server: any
```

Comment:
Is it possible to replace any with something meaningful?

Reply:
this.server = express() > this is where the variable is getting initialized.  We can replace any with "express.Express" , will that is meaningful ? or should we keep it as any

Reply:
I suppose so

Reply:
Changes done